src/views/home/Home.tsx

```diff
@@ -11,20 +15,28 @@ type Props = {
   currencies: Currency[];
   selectedCurrency: Currency;
   products: FetchProductResponse[];
+
 };
 export const Home: FC<Props> = ({
   categories,
   currencies,
   selectedCategory,
   selectedCurrency,
   products,
+
 }) => {
+ // const dispatch = useDispatch();
```

Comment:
If this comment serves a purpose, it will be great if there was a comment explaining why this is the case.